app/views/activity_insight_oa_workflow/wrong_file_version_curation/index.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="container pt-4 pb-5">
+  <%= link_to '<< Back', activity_insight_oa_workflow_path %>
+  <h1 class="pb-3 pt-2">Publications with Wrong File Version</h1>
+  <table class="table">
+    <thead class='card-header'>
+      <tr>
+        <th>Author</th>
+        <th>Author Batch Email</th>
+        <th>Publication Email</th>
+        <th>Email Last Sent On</th>
+        <th>Title</th>
+        <th>Preferred Version</th>
+        <th>File Version</th>
+        <th>Download File</th>
+      </tr>
+    </thead>
+    <tbody class='card-body'>
+
+    <% authors_grouped = @publications.group_by { |p| p.activity_insight_oa_files.first.user_id } %>
+    <% authors_grouped.each do |author, publications| %>
+      <% publications.each do |p, index| %>
+          <tr id="<%= dom_id(p) %>">
+          <td rowspan="<%= p.activity_insight_oa_files.count %>">
+            <% if publications.index(p) == 0 %>
+              <%= User.find(p.activity_insight_oa_files.first.user_id).webaccess_id %>
+            <% end %>
+          </td>
+          <td rowspan="<%= p.activity_insight_oa_files.count %>">
+            <% if publications.index(p) == 0 %>
+                <%= button_to "Send Batch Email", activity_insight_oa_workflow_wrong_file_version_email_path, params: {publications: publications} %>
+            <% end %>
+          </td>
```

Comment:
We move some things around here and get the table nesting a little more aligned:

```suggestion
      <tr id="<%= dom_id(User.find(author)) %>">
      <td rowspan="<%= publications.collect(&:activity_insight_oa_files).flatten.count %>">
        <%= User.find(author).webaccess_id %>
      </td>
      <td rowspan="<%= publications.collect(&:activity_insight_oa_files).flatten.count %>">
       <%= button_to "Send Batch Email", activity_insight_oa_workflow_wrong_file_version_email_path, params: {publications: publications} %>
      </td>
      <% publications.each do |p| %>
```

This gives us something like this:

![Screenshot 2023-09-06 at 3 25 18 PM](https://github.com/psu-libraries/researcher-metadata/assets/32677188/1181a674-c668-46dd-b725-8f5f04821002)

Also, there's the chance we end up with multiple files for a single publication where the authors are different.  Since we select the first OA file for every publication to base the author off of, and also do that in the email sending, I think we should be fine.